tests/installation/disable_grub_timeout.pm

```diff
@@ -55,7 +55,11 @@ sub run {
     # to go to the first tab
     send_key 'tab' unless match_has_tag 'inst-bootloader-settings-first_tab_highlighted';
 
-    my $bootloader_shortcut = (is_x86_64) ? 'alt-r' : 'alt-t';    # Seems now the bootloader shortcut is a different one
+    # Seems like difference between UEFI and Legacy for sle15sp5+.
+    my $bootloader_shortcut = (is_x86_64) ?
+      ((check_var('UEFI', '1')) ? 'alt-t' : 'alt-r')    # uefi on x86 has different behavior
+      : 'alt-t';    # t is the default for non x86
+
```

Comment:
howabout `my $bootloader_shortcut = ((is_x86_64) && !check_var('UEFI', '1') )?`

Reply:
I think I will stick to the code it's already written (more because of readability).